gpustack/schemas/clusters.py

```diff
@@ -64,6 +64,10 @@ class WorkerPoolCreate(WorkerPoolUpdate):
         default={}, sa_column=Column(pydantic_column_type(CloudOptions))
     )
     zone: Optional[str] = None
+    # instance_spec is for UI to store the instance_type's extended specifications for display.
+    instance_spec: Optional[Dict[str, Any]] = Field(
```

Comment:
How does it work, do you have examples?

Reply:
Current UI stores following informations:
```json
{"vram": "48 GiB", "vcpus": 8, "ram": "64 GiB", "bootDisk": "500 GiB", "label": "48 GiB VRAM / 8 vCPUs / 64 GiB RAM / 500 GiB Boot disk", "vendor": "nvidia"}
{"vram": "80 GiB", "vcpus": 20, "ram": "240 GiB", "bootDisk": "720 GiB", "scratchDisk": "5 TiB", "label": "80 GiB VRAM / 20 vCPUs / 240 GiB RAM / 720 GiB Boot disk / 5 TiB Scratch disk", "vendor": "nvidia", "description": "H100 GPU - 1X"}
```

Those informations are scraped from DO(cloud provider) with `instance_type` and stored in `instance_spec` for UI to show the details of this instance spec.